build.gradle

```diff
@@ -398,6 +398,7 @@ task testIntegration(type: Test) {
         maxRetries = 3
         maxFailures = 15
     }
+    failFast = true
```

Comment:
Are we sure we want to keep failFast?  Does that mean it won't run tests after the first one fails?

Reply:
I've removed it!